core/build.gradle.kts

```diff
@@ -0,0 +1,90 @@
+val kotlinVersion: String by project
+
+plugins {
+    kotlin("multiplatform") version "1.3.60-eap-76"
```

Comment:
Let's avoid duplicating `kotlinVersion` as a literal here.

Reply:
That's impossible, because this part of Gradle is executed earlier than resolves properties with Kotlin DSL. Discussed this with @ilmat192, this is known limitation.

Reply:
Since version 5.6 Gradle allows reading plugin versions from `gradle.properties` in the `pluginManagement` block. See https://docs.gradle.org/5.6/userguide/plugins.html#sec:plugin_version_management.

Reply:
That's clearly possible by specifying buildscript classpath in the root project, as it is done in kotlinx.collections.immutable. Feel free to leave it for me, though.